### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by kgaut**
Some strings translated / cleaned in French.

### CODE REVIEW

File: config/translations/console.fr.yml [hunk 1]
```diff
@@ -92,7 +92,7 @@ commands:
             description: 'Éditer la configuration sélectionnée.'
```
Comment (OriPekelman) [inline config/translations/console.fr.yml]: "Editer" could mean "publish" prefer the form "Modifier la configuration"

File: config/translations/console.fr.yml [hunk 2]
```diff
@@ -162,7 +162,7 @@ commands:
                 entity-name: 'Entrez le nom de l''entitée'
```
Comment (OriPekelman) [inline config/translations/console.fr.yml]: "entité" does not take a final e

File: config/translations/console.fr.yml [hunk 3]
```diff
@@ -199,41 +199,41 @@ commands:
...
+                machine-name: 'Entrez le nom machine du thème'
+                theme-path: 'Entrez le dossier du thème'
+                description: 'Entrez la description du thème'
+                core: 'Entrez la version du core de Drupal'
+                package: 'Entrez le nom du paquet <info>package</info>'
+                dependencies: 'Voulez-vous ajouter des dépendances pour le thème ?'
+                invalid-theme: 'Le thème %s n''est pas valide'
             warnings:
-                module-unavailable: 'Warning The following modules are not already available in your local environment "%s"'
+                module-unavailable: 'Attention, les modules suivants ne sont pas disponible dans votre installation locale : "%s"'
```
Comment (OriPekelman) [inline config/translations/console.fr.yml]: "disponible" should be in plural form "disponibles"

File: config/translations/console.fr.yml [hunk 4]
```diff
@@ -606,20 +606,20 @@ commands:
                 invalid-user: 'Invalid user id %s'
     yaml:
         merge:
-            description: 'Merge one or more YAML files in a new YAML file. Latest values are preserved.'
+            description: 'Fusionne un ou plusieurs fichiers YAML en un nouveau. Les valeurs les plus récentes sont concervées.'
```
Comment (OriPekelman) [inline config/translations/console.fr.yml]: "concervées" should be written "conservées"

File: config/translations/console.fr.yml [hunk 5]
```diff
@@ -606,20 +606,20 @@ commands:
...
+                file-already-added: 'Ce fichier est déjà présent dans la fusion'
             messages:
-                wrong-parse: 'Yaml file "%s" is empty or doesn''t exist the file.'
-                error-parsing: 'An error occurs during parsing of Yaml file %s.'
-                two-files-required: 'At least to files are required.'
-                error-generating: 'Error generating merged YAML.'
-                error-writing: 'Error writing merged YAML file.'
-                merged: 'New file %s was created sucessfully after merge Yaml files.'
+                wrong-parse: 'Le fichier "%s" est vide ou n''existe pas.'
+                error-parsing: 'Une erreur est survenue lors du parsage du fichier %s.'
```
Comment (OriPekelman) [inline config/translations/console.fr.yml]: prefer "analyse du fichier", "parsage" is not an accepted French word.

File: config/translations/console.fr.yml [hunk 6]
```diff
@@ -180,18 +180,18 @@ commands:
...
                 description: 'Description du module'
-                core: 'Version de core.'
-                package: 'Paquet Module'
+                core: 'Version de core'
+                package: 'Paquet (<info>package</info>) du module'
                 controller: 'Contrôleur par défaut'
-                composer: 'Add a composer.json file'
-                dependencies: 'Module dependencies (i.e. context, galleria, panels)'
+                composer: 'Ajouter un fichier composer.json'
+                dependencies: 'Dépendences du modules (par exemple : context, galleria, panels)'
```
Comment (jygastaud) [inline config/translations/console.fr.yml]: "Dépendences" should be "Dépendance" with a "a"
"modules" should not take a "s"

File: config/translations/console.fr.yml [hunk 7]
```diff
@@ -199,41 +199,41 @@ commands:
...
                 controller: 'Voulez-vous générer un contrôleur par défaut'
-                composer: 'Do you want to add a composer.json file to your module'
-                dependencies: 'Would you like to add module depencencies'
-                test: 'Voulez-vous générer une classe de test unitaire'
+                composer: 'Voulez-vous générer un fichier composer.json file pour votre module ?'
+                dependencies: 'Voulez-vous ajouter des dépendances à votre module ?'
+                test: 'Voulez-vous générer une classe de test unitaire ?'
             warnings:
-                module-unavailable: 'Warning The following modules are not already available in your local environment "%s"'
+                module-unavailable: 'Attention, les modules suivants ne sont pas disponible dans votre installation locale : "%s"'
```
Comment (jygastaud) [inline config/translations/console.fr.yml]: disponible should be in plural form => disponibles

File: config/translations/console.fr.yml [hunk 8]
```diff
@@ -199,41 +199,41 @@ commands:
...
-                description: 'Theme description'
-                core: 'Core version'
-                package: 'Theme package'
-                composer: 'Add a composer.json file'
-                base_theme: 'Base theme (i.e. classy, seven)'
+                theme: 'Nom du thème'
+                machine-name: 'Le nom machine (minuscules et underscore _ seulement)'
+                theme-path: 'Dossier du thème'
+                description: 'Description du thème'
+                core: 'Version du core'
```
Comment (jygastaud) [inline config/translations/console.fr.yml]: Should we translate "core" into "cœur" ?

Comment (OriPekelman) [inline config/translations/console.fr.yml]: I believe it is customary in the French  Drupal community to talk about "core". I would keep it as such.

Comment (jygastaud) [inline config/translations/console.fr.yml]: I tend to use both.

Looking at translation into Drupal French Localize I can see at least 3 variations : 
* core are keep as it
* core is translated as "cœur", which looks like majoritary
* core is translated as "noyau"

Though, we can ask Localize Team for that ?

**Pull request merged**